pytorch_to_returnn/naming/naming.py

```diff
@@ -325,8 +325,8 @@ def register_input(self, tensor: _types.Tensor, returnn_data: Data) -> Data:
       feature_dim_axis=returnn_data.feature_dim_axis_or_unspecified,
       available_for_inference=True)
     entry.returnn_axis_from_torch_axis = {i: i for i in range(returnn_data.batch_ndim)}
-    if entry.returnn_data.have_batch_axis():
-      tensor.shape[entry.returnn_data.batch_dim_axis].is_batch_dim = True
+    for dim in range(returnn_data.batch_ndim):
```

Comment:
Better call it `axis`, for consistency.
```suggestion
    for axis in range(returnn_data.batch_ndim):
```